### ISSUE TITLE
Experiment Setting

### ISSUE THREAD

**Issue by wsgdrfz** [wsgdrfz/Scale-invariant-CNNs#7]
## MNIST
- Baseline 1: Standard CNN with the training set MNIST
- Baseline 2: Standard CNN with the training set MNIST+MNIST-SCALE (SHUFFLE)
- SI-Conv with scale (1.0, 3.0, 6) and training set MNIST
- SS-CNN with scale () and training set MNIST
- Training size: 1k, 2k, 4k, 6k, 8k, 10k
- Testing set: MNIST-Scale 10k
- Validation set: MNIST-Scale 2k
- Epoch: 30
- Measurements: Accuracy, number of epochs, training time

**Comment by wsgdrfz** [comment]
## FMNIST
- Baseline: Standard CNN  with the training set MNIST+MNIST-SCALE (SHUFFLE)
- SI-Conv with scale (1.0,1.5,5) and training set MNIST
- SS-CNN with scale () and training set MNIST
- Training size: 1k, 2k, 4k, 6k, 8k, 10k
- Epoch: 30/50 (need experiment)
- Testing set: MNIST-scale
Problem: data argumentation should add the size of the set??

**Comment by wsgdrfz** [comment]
## Oral Cancer 
- Baseline: Standard CNN with the training set ORAL+ORAL-SCALE
- SI-CONV with scale (0.5,1.0,5) and training set ORAL
- SS-CNN with scale () and training set ORAL
- SS-CNN + BLUR POOL
- SI-CONV + BLUR POOL
- Training size: 10k, 30k, 50k, 70k
- Epoch: 30/50 (need experiment)
- Testing set: ORAL-SCALE 10K
